mlbench_core/dataset/linearmodels/pytorch/dataloader.py

```diff
@@ -208,9 +226,35 @@ def get_dataset_info(name):
     return stats[0]
 
 
-def load_libsvm_lmdb(name, lmdb_path):
-    stats = get_dataset_info(name)
-    dataset = IMDBPT(lmdb_path, transform=maybe_transform_sparse(stats),
-                     target_transform=None, is_image=False)
-    return dataset
+def load_and_download_lmdb(name, data_type, dataset_dir):
```

Comment:
this should be part of `LMDBDataset` and done in the `__init__` with a `download=True` flag to be consistent with the other dataloaders